news/1626.feat

```diff
@@ -0,0 +1,2 @@
+Apply a cache ruleset to the /@querystring-search endpoint.
```

Comment:
@ericof isn't the file extension .feature? I will check after merging this one. No action is needed. Just FYI.